suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -424,10 +427,12 @@ const discoverNetworkBatchThunk = createThunk(
             deviceState,
             round = 1,
             network,
+            accountType,
         }: {
             deviceState: string;
             round?: number;
-            network: NetworkCompatible;
+            network: Network;
+            accountType: AccountType;
```

Comment:
this function relied on getting a specific `network` for given `accountType`, now it's normal `network` and `accountType` passed alongside.